src/components/Onboarding/ChartChooser.tsx

```diff
@@ -51,22 +60,69 @@ export default function ChartChooser(props: ChartChooserProps) {
     props.onChartSelected(spec);
   }
 
+  function handleKeyDown(e: React.KeyboardEvent<HTMLInputElement>) {
+    switch (e.key) {
+      case 'Enter':
+        e.preventDefault();
+        e.stopPropagation();
+        displayChart();
+        break;
+      case 'ArrowRight':
+        if (selectedIndex < suggestedGraphs.length - 1) {
+          setSelectedIndex(selectedIndex + 1);
+        }
+        break;
+      case 'ArrowLeft':
+        if (selectedIndex !== 0) {
+          setSelectedIndex(selectedIndex - 1);
+        }
+        break;
+    }
+  }
```

Comment:
Maybe we should abstract this into a hook, since its used in 2 different files. To keep it DRY

Reply:
Although they look similar, keydowns are used differently in ChartChooser.tsx and ColumnScreen.tsx. And, ColumnScreen handles more keys than the other.